src/lib/games/queries.ts

```diff
@@ -233,8 +233,38 @@ export async function getGameResultsData(tx: DBOrTransation, gameId: string) {
     where: cmp.eq(schema.questions.id, game.question_id),
   })
 
+  const playersWithFinalCode = await Promise.all(
+    playerGameSessions.map(async (playerSession) => {
+      const [latestSubmittedChatHistoryItem] = await tx
```

Comment:
kinda feels sus making a query for each player in the game but idk that might be fine

Reply:
should be fine for now - need to add in metrics for now long some functions take at some point tho